### PR TITLE
Support yolo-v4 tf2 for object detection demo

### PR DESCRIPTION
**Pull request by akorobeinikov**

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
are there issues with yolo-v4 support in python object_detection_demo?

**Comment by akorobeinikov** [comment]
> are there issues with yolo-v4 support in python object_detection_demo?

Yes. Output names for `yolo-v4-tf` looks like `StatefulPartitionedCall/model/conv2d_<number>/BiasAdd/Add`, but no `conv2d_<number>/BiasAdd/Add`. The demo requires names to start with `conv`.

**Comment by vladimir-dudnik** [comment]
missed in the release?

**Comment by akorobeinikov** [comment]
> missed in the release?

Yes, missed

**Review by eizamaliev — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
@akorobeinikov @eizamaliev line `if name.startswith('conv'):` was added by @dkurt to distinguish yolo-v4-tiny model. were there any changes in MO regarding yolo-v4-tiny model, which makes this line not needed?

**Comment by dkurt** [comment]
@vladimir-dudnik , seems like that because I've verified the demo when added this check.

File: demos/common/python/models/yolo.py [hunk 1]
```diff
@@ -245,10 +245,9 @@ def _get_output_info(self):
 
         output_info = {}
         for i, (name, layer) in enumerate(outputs):
-            if name.startswith('conv'):
```
Comment (dkurt) [inline demos/common/python/models/yolo.py:248]: Maybe let's replace to `if name.find('conv') != -1` ?

**Review by dkurt — APPROVED** [review]

**Comment by vladimir-dudnik** [comment]
closing this, the same change will come from release branch

**Pull request closed without merging**